mindmaps-titan-factory/pom.xml

```diff
@@ -114,6 +114,10 @@
                     <groupId>org.eclipse.jetty.orbit</groupId>
                     <artifactId>javax.servlet</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
If we are now excluding this everywhere I can remove the exclusions from the upper poms. I.e. `mm-test/titan-test`